raven/raven.go

```diff
@@ -268,3 +306,29 @@ func (T *transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	defer timer.Stop()
 	return T.httpTransport.RoundTrip(req)
 }
+
+type EventEncoder interface {
```

Comment:
yeah that would make sense. Let's make it a standalone function for now, no need for a method at this point.